eagle-server/src/main/webapp/app/dev/public/js/ctrls/alertEditCtrl.js

```diff
@@ -304,6 +304,15 @@
 			return false;
 		};
 
+
+		$scope.checkPolicyName = function () {
+			if($scope.policy.name.length > 50) {
+				return "length should less than 50";
```

Comment:
This message will combine with "Name". Finally, user will see the message "Name length should less than 50".